sdkconfig

```diff
@@ -143,14 +146,14 @@ CONFIG_NIMBLE_CPP_ENABLE_ADVERTISMENT_TYPE_TEXT=y
 #
 # Serial flasher config
 #
-CONFIG_ESPTOOLPY_PORT="/dev/cu.usbserial-*"
-CONFIG_ESPTOOLPY_BAUD_115200B=
+CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
```

Comment:
is this the port that's exposed in Docker??

Reply:
That's the default value from the menuconfig tool. I think it works if you're on linux. There's an issue in docker on macOS and windows that doesn't allow the container to flash directly (https://github.com/docker/for-mac/issues/900). Thats why I'm still using the esptool.py for flashing. Right now you can call `./build.sh flash -p <espPort>`.